### ISSUE TITLE
[.NET MAUI] Failure to build project with spaces in path

### ISSUE THREAD

**Issue by PramodKumarHK89** [msintuneappsdk/ms-intune-app-sdk-android#180]
# Intune Android App SDK for .NET MAUI Issue

## Summary

Unable to build my project on latest Visual Studio 2022 on my windows machine. I'm cloning the project from https://github.com/gustavoja/Intune.MAM.NET7

## Details

Microsoft.Intune.Maui.Essentials.android` Version: beta2**
Operating System - Windows 11
Visual Studio Version
[Diagnostic_Output.zip](https://github.com/msintuneappsdk/ms-intune-app-sdk-android/files/12617104/Diagnostic_Output.zip)
 2022 latest update

## Logs

For **build issues**, please include:

- MSBuild Logs
INCLUDED

For **runtime issues**, please include:

- MSBuild Logs - INCLCUDED
- CP Logs

## NuGet Dependencies

If your app includes third-party NuGet libraries, please list them here:

- NA

## Questions to Ask Before Submission

1. Does the app build and launch without including the `Microsoft.Intune.Maui.Essentials.android` package? Yes
2. Does the issue reproduce if you rebuild after deleting the `bin` and `obj` output folders? Yes
3. Have you checked the [.NET MAUI repository](https://github.com/dotnet/maui/issues) for similar issues? If so, please follow the guidance there or add that information to this issue. Nope
4. Are you using the latest `Microsoft.Intune.Maui.Essentials.android` NuGet package version? Yes

**Comment by MikeP0911** [comment]
Did you change the default values of intune settings in the project?

**Comment by rygo-msft** [comment]
```
2>    Running command:
2>      Command: C:\Users\pramkum\.nuget\packages\microsoft.intune.maui.essentials.android\9.5.2-beta2\build\netstandard2.0\BuildTool\bin\BuildToolCLI.bat
2>      Arguments: --input C:\Official\IntuneTraining\Session4 - MAUI-Xamarin - IntuneAppSDK\MAUISample\Intune.MAM.NET7.Droid\obj\Debug\net7.0-android33.0\intune\build-tool\remap\input.txt --output C:\Official\IntuneTraining\Session4 - MAUI-Xamarin - IntuneAppSDK\MAUISample\Intune.MAM.NET7.Droid\obj\Debug\net7.0-android33.0\intune\build-tool\remap\output.txt --excludeClasses mono.MonoRuntimeProvider;androidx.startup.InitializationProvider --classpath "C:\Program Files (x86)\Android\android-sdk\platforms\android-33\android.jar;C:\Users\pramkum\.nuget\packages\microsoft.intune.maui.essentials.android\9.5.2-beta2\build\netstandard2.0\Resources\classes.jar" --report C:\Official\IntuneTraining\Session4 - MAUI-Xamarin - IntuneAppSDK\MAUISample\Intune.MAM.NET7.Droid\obj\Debug\net7.0-android33.0\intune\build-tool\remap\report --xamarin
2>    File: C:\Official\IntuneTraining\Session4 not found.
```

This looks like an issue related to spaces on file paths not being handled properly. As a work around, you should be able to use a project directory without spaces.

**Comment by PramodKumarHK89** [comment]
@rygo-msft . Yes, it is. 

Issue is resolved after copying the project into a path without space

**Comment by Archit-Pikle** [comment]
Fix is shipped in version 9.7.1-beta
